OpenOversight/app/templates/leaderboard.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container" role="main">
+
+
+  <div class="page-header">
+    <h1>OpenOversight Volunteer Leaderboard</h1>
+  </div>
+
+  <div class="row">
```

Comment:
Was on the fence about this but in fa48210c0d431ecd62d342bf31144903b0912498 have modified this to require login.